TeX_files/chapter02.tex

```diff
@@ -190,7 +190,7 @@ \section{Analysis of Transmission line}
 \frac{d^{2}I}{dx^{2}} = - (G + j\omega C)\frac{dv}{dx}
 \label{eqn:delta2i}
 \end{align}
-Substituting equation \ref{eqn:delta2v} into \ref{eqn:delta2i} we have,
+Substituting equation \ref{eqn:deltai} into \ref{eqn:delta2i} we have,
```

Comment:
Please review this change, I am not sure why you have done so. Shouldn't one equation be substituted into another, either voltage in current or current in voltage?

Reply:
Should be this instead

```suggestion
Substituting equation \ref{eqn:deltai} into \ref{eqn:delta2v} we have,
```